code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -108,6 +108,19 @@
 				new /obj/item/weapon/soap/syndie(src)
 
 
+			if("fakeops")//
+				new /obj/item/clothing/head/helmet/space/syndicate(src)
+				new /obj/item/clothing/suit/space/syndicate(src)
+				new /obj/item/weapon/gun/projectile/automatic/c20r(src)
+				new /obj/item/ammo_storage/magazine/a12mm/ops(src)
+				new /obj/item/weapon/card/id/syndicate(src)
+				new /obj/item/weapon/implanter/explosive(src)
+				new /obj/item/clothing/shoes/magboots/syndie(src)
+				new /obj/item/clothing/under/syndicate(src)
+				new /obj/item/clothing/mask/gas/syndicate(src)
+				new /obj/item/clothing/gloves/combat(src)
```

Comment:
You just won't be able to put it all back in after you take it out.
Not that you could anyway, considering some of the stuff has a higher `w_class` than a box can hold.